Explorer/Assets/DCL/LOD/Settings/ILODSettingsAsset.cs

```diff
@@ -10,6 +10,7 @@ public interface ILODSettingsAsset
         //Threshold for bucket partition (inclusive)
         int[] LodPartitionBucketThresholds { get;  }
         int SDK7LodThreshold { get; set;  }
+        float TimeToChangeToLod { get; set; }
```

Comment:
Please write documentation to provide some context about this new field, why is it necessary, what is its purpose or who uses it for what.